symplyphysics/laws/optics/abbe_invariant.py

```diff
@@ -0,0 +1,41 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+    validate_input, validate_output, dimensionless)
+
+# Description
+## The Abbe invariant is an optical characteristic of the refractive curve of the surface
+
+# Law: Q = n * ((1 / d) - (1 / R))
+# Where:
+## n - surface refraction index
+## d - distance to surface
+## R - surface curvature radius
```

Comment:
Does the sign change for convex and concave surface?

Reply:
It was meant that the surface into which the ray enters is convex. I will indicate this at the next fix.